src/applications/personalization/profile/components/ProfileContext.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { createContext } from 'react';
+import { useSelector } from 'react-redux';
+import { PROFILE_TOGGLES } from '../constants';
+import { selectProfileToggles } from '../selectors';
+
+const initialContext = {
+  toggles: { loading: true, ...PROFILE_TOGGLES },
+};
+
+export const ProfileContext = createContext(initialContext);
+
+export const ProfileContextWrapper = ({ children }) => {
+  const toggles = useSelector(selectProfileToggles);
+
+  return (
+    <ProfileContext.Provider
+      value={{
+        ...initialContext,
+        ...{ toggles },
+      }}
+    >
+      {children}
+    </ProfileContext.Provider>
+  );
+};
```

Comment:
This is currently unused. 

I created it as a way to provide a common 'container' with context to supply shared values or functions across any profile component. I ran into issues with cyclic dependencies when trying to add route configuration values to this, so I removed that and just put the feature toggles in for now. Its a WIP I guess.